packages/contracts-bedrock/contracts/test/OptimismMintableERC20.t.sol

```diff
@@ -64,4 +66,24 @@ contract OptimismMintableERC20_Test is Bridge_Initializer {
         vm.prank(address(alice));
         L2Token.burn(alice, 100);
     }
+
+    function test_erc165_supportsInterface() external {
+        // The assertEq calls in this test are comparing the manual calculation of the iface,
+        // with what is returned by the solidity's type().interfaceId, just to be safe.
+        bytes4 iface1 = bytes4(keccak256("supportsInterface(bytes4)"));
+        assertEq(iface1, type(IERC165).interfaceId);
+        assert(L2Token.supportsInterface(iface1));
+        emit log_bytes32(bytes32(iface1));
```

Comment:
did you mean to leave in these log statements?

Reply:
No, removed them.